rabbit-escape-engine/src/rabbitescape/engine/textworld/LineProcessor.java

```diff
@@ -125,6 +127,16 @@ public LineProcessor(
 
         process( variantGen );
     }
+    
+    public Comment[] getComments()
+    {
+        Comment [] ret = new Comment[comments.size()];
+        for ( int i = 0; i < comments.size(); i++ )
+        {
+            ret[i] = comments.get( i );
+        }
+        return ret;
+    }
 
```

Comment:
Will Collections.toArray do this for us? (Not sure we are allowed to use it in Android API level 8 either.) If not I'm fairly sure I wrote a Util method for this.